pkg/opssight/creater_test.go

```diff
@@ -153,13 +153,13 @@ func validateConfigMaps(t *testing.T, configMaps []*components.ConfigMap, opssig
 
 	for _, cm := range configMaps {
 		actualConfigMap := expectedConfigMaps[cm.GetName()]
-		if !strings.EqualFold(cm.GetObj().Name, actualConfigMap.name) {
-			t.Errorf("config map name is not equal. Expected: %s, Actual: %s", actualConfigMap.name, cm.GetObj().Data)
+		if !strings.EqualFold(cm.ConfigMap.Name, actualConfigMap.name) {
```

Comment:
Since this test case functions are commented, it is not called now. I will create a separate PR to uncomment those functions and test the OpsSight orchestration